test/integration/helpers.sh

```diff
@@ -292,6 +292,7 @@ function start_sim() {
         case "$TPM2_SIM" in
             *swtpm) "$TPM2_SIM" socket --tpm2 --server port="$tpm2_sim_port" \
                                        --ctrl type=tcp,port="$tpm2_sim_cmd_port" \
+                                       --seccomp action="none" \
```

Comment:
@idesai

Reply:
@williamcroberts @idesai should I only disable seccomp for multi arch builds or for all builds?